### DISCUSSION TITLE
oioioi

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#450] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
how do i change the dns with the wifi password

**Comment by Aka-but-nice** [comment]
<a href="https://sipe.glitch.me/">this</a>

**Reply by Darcii** [reply]
bro what?

**Reply by rwg100** [reply]
if only glitch.me wasnt blocked

**Comment by Calamitou** [comment]
dude that doesn't show u how to change the dns

**Comment by piplup7575** [comment, marked as answer]
Hey!
If you follow the directions on the readme, under the ltbeef tab, you can see you can change the dns under the network settings tab in the chromeos settings.
see the attached screenshot
![image](https://user-images.githubusercontent.com/88395302/215272726-c297aa44-28cc-4c35-949e-3c83e1966504.png)

**Reply by Darcii** [reply]
dang my school somehow has that blocked It wont allow me to edit it.
